velox/common/memory/Memory.cpp

```diff
@@ -59,7 +59,8 @@ std::unique_ptr<MemoryArbitrator> createArbitrator(
            std::min(options.arbitratorCapacity, options.allocatorCapacity),
        .memoryPoolTransferCapacity = options.memoryPoolTransferCapacity,
        .memoryReclaimWaitMs = options.memoryReclaimWaitMs,
-       .arbitrationStateCheckCb = options.arbitrationStateCheckCb});
```

Comment:
Can you help add error message if memory pool leak check is disabled like we do for memory arbitrator? Thanks!